tools/pipelines/templates/build-npm-package.yml

```diff
@@ -261,20 +268,37 @@ stages:
               targetType: 'inline'
               workingDirectory: ${{ parameters.buildDirectory }}
               script: |
-                mkdir $(Build.ArtifactStagingDirectory)/pack/
+                mkdir $(Build.ArtifactStagingDirectory)/pack-scoped/
+                mkdir $(Build.ArtifactStagingDirectory)/pack-non-scoped/
```

Comment:
this folder gets created even when there are no unscoped packages.  i'd suspect at a minimum that means we end up with an empty folder artifact for server, etc., but do you know how that affects the publish step?  would it be better not to create this when there are no unscoped packages specified?

Reply:
When the folder is empty, no artifact gets uploaded due to devops' own logic (https://dev.azure.com/fluidframework/internal/_build/results?buildId=34161&view=results). Investigating how this affects the publish step and I'll add in a len check here to avoid this folder creation

Reply:
Updated to now set a `publishNonScopedPackages` variable in `include-vars.yml`. The folder creation and additional pack, artifact upload, and publishing steps will only run when this is set to true and the `nonScopedPackages` parameter has some values